kedro/contrib/io/pyspark/spark_data_set.py

```diff
@@ -62,6 +62,10 @@ def _split_filepath(filepath: str) -> Tuple[str, str]:
     return "", split_[0]
 
 
+def _strip_dbfs_prefix(path: str) -> str:
+    return path[5:] if path.startswith("/dbfs") else path
```

Comment:
To make it more obvious where the magic 5 is coming from.
```suggestion
    prefix = "/dbfs"
    return path[len(prefix):] if path.startswith(prefix) else path
```